src/remediations/controller.write.js

```diff
@@ -170,7 +170,24 @@ exports.patch = errors.async(async function (req, res) {
         }
 
         if (name) {
+            const existing_remediation = await db.remediation.findOne({
+                attributes: ['id'],
+                where: { created_by: username, name: name },
+            }, {
+                transaction
+            });
+
+            // Do not allow users to use the same name for more than one remediation
+            if (existing_remediation) {
+                throw new errors.BadRequest('DUPLICATE_NAME', `Remediation name "${name}" for user "${username}" has already been taken`);
+            }
+
             remediation.name = name;
+        } 
+
+        // Do you not allow users to update a remediation name to empty string or null
+        if (name === "" || name === null) {
+            throw new errors.BadRequest('INVALID_NAME', "Remediation name cannot be empty"); 
         }
```

Comment:
What about merging these two conditions for the `name` together?

Reply:
`if (!_.is_empty(name))` would be more consistent with the rest of the code in this file.

Reply:
Yep, good idea I'll use this! Do you mean `_.isEmpty()` though?

Reply:
Whoops!   Yeah, `_.isEmpty()` 😬